### ISSUE TITLE
Upgrade mandoc to 1.14.5

### ISSUE THREAD

**Issue by zhsj** [Debian/debiman#127]
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=983010

**Comment by stapelberg** [comment]
Unfortunately, [mandoc 1.14.5 contains a bug](https://inbox.vuxu.org/mandoc-discuss/CANnVG6mHkSnOTFXDnvGUJDWwd7JmqG07uPtJbxeO3O=PESYm-g@mail.gmail.com/) which makes it segfault on some of our pages.

The bug was fixed in 2019, but there hasn’t been a new release since.

We’ll either need to wait for a new release (I asked multiple times for one), or use a patched or from-HEAD version of mandoc, which I would generally like to avoid.

**Comment by nthykier** [comment]
Hi,

https://mandoc.bsd.lv/ lists " current version 1.14.6 (September 23, 2021) " now, so maybe this can be (or is already) fixed now?

**Comment by stapelberg** [comment]
Yeah. I mean, we’re *again* waiting for a new upstream release, and have been running a patched version of mandoc for years, so the situation is still pretty bad, but the specific bug that this issue was tracking has been fixed.
